Firebase/Core/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Unreleased
 
+# 2019-03-19 -- v5.4.0 -- M45
+- [changed] Allow Bundle IDs that have a valid prefix to enable richer extension support. (#2515)
+- [changed] Convert FIRLogger to use the recommended os_log for logging via GoogleUtilities.
+  See https://developer.apple.com/documentation/os/logging?language=objc.
```

Comment:
Nit: did you want to add PR number(s) here?

Reply:
Please don’t do this indiscriminately.

_Issue_ numbers are useful in release notes because they help affected users understand that a fix is for them. Unaffected users might go there to understand the context of the change more deeply.

PR numbers are just showing how the sausage is made.

A few cases where it might be appropriate to include PR numbers:
  * PRs from third parties elevate that acknowledge their contribution
  * PRs where there’s been a meaningful interaction where someone outside the team has interacted with it

Reply:
Please don’t do this indiscriminately.

_Issue_ numbers are useful in release notes because they help affected users understand that a fix is for them. Unaffected users might go there to understand the context of the change more deeply.

PR numbers are just showing how the sausage is made.

A few cases where it might be appropriate to include PR numbers:
  * PRs from third parties elevate that acknowledge their contribution
  * PRs where there’s been a meaningful interaction where someone outside the team has interacted with it

Reply:
Makes sense to me, thanks @wilhuff for the thoughtful note.

Do you think there's a difference of including them here vs including them in the release notes on the Firebase site? I can see that's a strong case for not including them on the Firebase site but including them here could be helpful for those more curious since they're already in the repo itself.

I don't have super strong feelings about including them here, but wasn't sure if they'd cause much harm.

Reply:
In this case, there was a series of PR's, so it would be difficult to associate even if we wanted to.

Reply:
Anyone sufficiently curious can `git log master` so our goal should be to help less interested parties establish context on changes without unnecessarily inflicting the development process on them.